<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>im_scissors</class>
 <widget class="QDialog" name="im_scissors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>928</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_form">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>General information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_questionaire">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Handedness questionnaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>What is your age?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>What is your biological sex?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_occupation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>What is your occupation and expertise?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_exp_microsurg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Experience in microsurgery (years)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="age">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sex_cb">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="occupation">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="exp_microsurg">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tweezers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Which hand do you use to handle the tweezers?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tweezer_cb">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_needleholder">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Which hand do you use to handle the needle holder?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="needleholder_cb">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_anastomosis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Are you familiar with anastomosis?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="anastomosis_cb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nbAnastomosis">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How often do you perform an anastomosis (per year)?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nbAnastomosis_cb">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nbCourses">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>227</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>How many training courses do you follow for anastomosis skills (per year)?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nbCourses_cb">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_confidenceAnastomosis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>How confident are you with anastomosis (0: not confident at all -&gt; 5: highly confident)?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="confidenceAnastomosis_cb">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_subjectnb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject number </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="subjectnb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="date">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_starttime">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Start time</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="starttime">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_questionaire_description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>1021</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please indicate which hand you prefer for each of the following activities. &lt;span style=&quot; font-weight:600;&quot;&gt;If sometime you use the other hand select &amp;quot;&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Usually &lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[Lelf or Right]&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>911</width>
     <height>572</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="im_drawing">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_drawing.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_broom">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;8. Using a broom &lt;/p&gt;&lt;p&gt;(dominant hand)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_drawing">
      <property name="text">
       <string>2. Drawing</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="ddm_writing"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="ddm_throwing"/>
    </item>
    <item row="2" column="6">
     <widget class="QComboBox" name="ddm_broom"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_throwing">
      <property name="text">
       <string>3. Throwing</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="im_teeth">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_teeth.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="ddm_drawing"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_scissors">
      <property name="text">
       <string>4. Using scissors</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="im_throwing">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_throwing.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_teeth">
      <property name="text">
       <string>5. Brushing teeth</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="im_writing">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_writing.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="im_match">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_match.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="im_broom">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_broom.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="im_jar">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_jar.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_writing">
      <property name="text">
       <string>1. Writing</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QComboBox" name="ddm_match"/>
    </item>
    <item row="1" column="6">
     <widget class="QComboBox" name="ddm_spoon"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_knife">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;6. Using knife&lt;/p&gt;&lt;p&gt;    (without fork)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_spoon">
      <property name="text">
       <string>7. Using a spoon</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="ddm_knife"/>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="im_spoon">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_spoon.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="im_scissor">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_scissors.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="ddm_teeth"/>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_match">
      <property name="text">
       <string>9. Striking a match</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="ddm_scissors"/>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_jar">
      <property name="text">
       <string>10. Opening a jar</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QComboBox" name="ddm_jar"/>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="im_knife">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images GUI/im_knife.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>890</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: green;
color: white;</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eye">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Which eye do you use when using only one?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="eye_cb">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
